### DISCUSSION TITLE
"resume from checkpoint" lead to CUDA out of memory

### DISCUSSION THREAD

**Discussion by Defiler24** [Lightning-AI/pytorch-lightning#11563] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
When I use “resume from checkpoint”, 
there is a “CUDA out of memory” problem, 
when using torch.load(), set "map location" to "cpu" can solve this problem, 
in "resume from checkpoint" scenario, what should I do?

**Comment by four4fish** [comment]
@Defiler24 Could you share which training strategy(plugin) are you using? Or could you share your code here

**Reply by Defiler24** [reply]
I did not set a specific strategy, the default should be DDP spawn.

    data  = DWheel(**vars(args))
    model = MWheel(**vars(args))

    trainer = Trainer(
                    max_epochs=480, 
                    gpus=8, 
                    sync_batchnorm=True,
                    resume_from_checkpoint="../version0/checkpoint/epoch=150.ckpt")
    trainer.fit(model, data)

**Comment by Defiler24** [comment, marked as answer]
I solved the problem after setting the strategy to 'ddp'.

**Comment by YueyangLiulyy** [comment]
I have exactly the same issue, the only difference is that my model is trained on single GPU. I did not specify any stratgey as well, is there any solution to solve it? Thanks
